src/pyhf/pdf.py

```diff
@@ -325,11 +332,10 @@ def _create_nominal_and_modifiers(self):
 
         self.mega_mods = mega_mods
 
-        tensorlib, _ = get_backend()
         thenom = default_backend.astensor(
             [mega_samples[s]['nom'] for s in self.config.samples]
         )
-        self.thenom = default_backend.reshape(
+        self._thenom = default_backend.reshape(
             thenom,
```

Comment:
Can we replace all instances of `thenom` and `_thennom` with something like `nominal_rate`? Having much more explicit names here helps a lot in reading the code.

Reply:
done

Reply:
Thanks! Snappy service. :)